superset/assets/src/dashboard/util/dashboardLayoutConverter.js

```diff
@@ -339,6 +512,7 @@ export default function(dashboard) {
       position = {
         ...position,
         code: form_data.code,
+        slice_name,
```

Comment:
what is this needed for? for markdown? is it necessary if you added it to the code? only asking because I don't think we'll be adding a title to the Markdown component.

Reply:
markdown is more than header, some markdowns are paragraphs of descriptions.

Reply:
oops, sorry I read this wrong and thought you were passing it within the markdown component meta data. I still agree adding it as header is the right way to go! 👍